src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.jsx

```diff
@@ -22,7 +22,6 @@ class GroupTagValues extends React.Component {
   static propTypes = {
     api: PropTypes.object,
     group: SentryTypes.Group.isRequired,
-    query: PropTypes.object,
```

Comment:
Should location be added the the prototypes?

Reply:
good catch! thanks